sanitizers/src/main/java/com/code_intelligence/jazzer/sanitizers/ScriptEngineInjection.java

```diff
@@ -0,0 +1,112 @@
+// Copyright 2022 Code Intelligence GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.code_intelligence.jazzer.sanitizers;
+
+import static java.util.Collections.unmodifiableSet;
+import static java.util.stream.Collectors.toSet;
+
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueCritical;
+import com.code_intelligence.jazzer.api.FuzzerSecurityIssueHigh;
+import com.code_intelligence.jazzer.api.HookType;
+import com.code_intelligence.jazzer.api.Jazzer;
+import com.code_intelligence.jazzer.api.MethodHook;
+import com.code_intelligence.jazzer.api.MethodHooks;
+import java.lang.invoke.MethodHandle;
+import java.util.Arrays;
+import java.util.Set;
+import java.util.stream.Stream;
+import java.io.IOException;
+import java.io.Reader;
+
+import javax.script.ScriptEngineManager;
+
+/**
+ * Detects Script Engine injection
+ *
+ * <p>
+ * The hooks in this class attempt to detect user input flowing into
+ * {@link javax.script.ScriptEngine.eval} that might lead to
+ * remote code executions depending on the scripting engine's capabilities.
+ * Before JDK 15, the Nashorn Engine
+ * was registered by default with ScriptEngineManager under several aliases,
+ * including "js". Nashorn allows
+ * access to JVM classes, for exmaple {@link java.lang.Runtime} allowing the
```

Comment:
```suggestion
 * access to JVM classes, for example {@link java.lang.Runtime} allowing the
```

You can also reflow the comment by turning it into a single line and then running the format script.